### PR TITLE
T10231: Add wgModerationOnlyInNamespaces to ManageWikiNamespaces.php

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3236,6 +3236,9 @@
 			NS_FILE
 		],
 	],
+	'wgModerationIgnoredInNamespaces'=> [
```
Comment (Reception123) [inline LocalSettings.php:3239]: why does wgModerationOnlyInNamespaces not also have an entry in LocalSettings.php?

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:3239]: It wasn't on LS.php before, that's why I added it via this commit.

**Comment by ugochimobi** [comment]
> This should not have an override in LocalSettings, and be added to ManageWikiNamespaces, not ManageWikiSettings.

Okay, sir, I'll move it to ManageWikiNamespaces and remove the override from LS.php.

**Comment by ugochimobi** [comment]
Should the `wgModerationOnlyInNamespaces` that was already in LS.php also be removed?

File: LocalSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply LocalSettings.php:3239]: ManageWikiNamespaces variables don't need LS variables. This one is already in MWN.

Comment (ugochimobi) [reply LocalSettings.php:3239]: But looks like UO said they should not be in LS.php at all if they're going to be added to managewikinamespaces https://github.com/miraheze/mw-config/pull/5053#issuecomment-1377647676

**Comment by Reception123** [comment]
@ugochimobi please check your PR, you accidentally deleted the whole ManageWikiSettings.php file.

**Comment by ugochimobi** [comment]
> @ugochimobi please check your PR, you accidentally deleted the whole ManageWikiSettings.php file.

Is there a way to revert the deletion?

**Comment by Reception123** [comment]
> > @ugochimobi please check your PR, you accidentally deleted the whole ManageWikiSettings.php file.
> 
> Is there a way to revert the deletion?

There are but I think you need GitHub command line. Probably easiest to just copy/paste the original one from [f5d73ed](https://github.com/miraheze/mw-config/pull/5053/commits/f5d73eddf09d28882f342ed9ec2a430eee271794)

**Comment by Universal-Omega** [comment]
Just open a new PR.

**Comment by ugochimobi** [comment]
> Just open a new PR.

I think I fixed it?

**Comment by ugochimobi** [comment]
We're good to go with merging the PR now.

**Comment by Reception123** [comment]
I'm afraid this still doesn't make sense as there's no wgModerationIgnoredInNamespaces in the ManageWikiNamespaces.php file, it's been completely removed.

**Review by Reception123 — APPROVED** [review]

File: ManageWikiNamespaces.php [hunk 2]
```diff
@@ -343,4 +343,15 @@
 		'help' => '',
 		'requires' => [],
 	],
+	'wgModerationOnlyInNamespaces' => [
+		'name' => 'Allow non-automoderated users to bypass moderation only in this namespace?',
```
Comment (edwardspec) [inline ManageWikiNamespaces.php:347]: That's not what it does. If `$wgModerationOnlyInNamespaces` is not `[]` (default), then moderation is disabled in all namespaces except those listed in `$wgModerationOnlyInNamespaces`.

**Pull request merged**